src/components/ProductCard/ProductCard.tsx

```diff
@@ -0,0 +1,40 @@
+import React, {FC} from 'react'
+import {StyleSheet, Text, TouchableOpacity} from 'react-native'
+import {Card} from 'react-native-elements'
+import {ProductModel} from '../../features/products'
+
+interface CardProps {
+  product: ProductModel
+  onClick: (id: string) => void
+}
+
+export const ProductCard: FC<CardProps> = ({product, onClick}) => {
+  return (
+    <TouchableOpacity onPress={() => onClick(product.id)}>
+      <Card>
+        <Card.Image source={require('../../assets/images/placeholder.jpeg')} />
+        <Card.Title style={styles.title}>{product.attributes.name}</Card.Title>
+        <Card.Divider />
+        <Text style={styles.description} numberOfLines={2}>
+          {product.attributes.description}
+        </Text>
+        <Card.Divider />
+        <Text style={styles.price}>{product.attributes.display_price}</Text>
+      </Card>
+    </TouchableOpacity>
+  )
+}
+
+const styles = StyleSheet.create({
```

Comment:
for now okay, but better keep style files separately from components

Reply:
Will be done, thanks